scripts/install

```diff
@@ -81,7 +80,7 @@ ynh_app_setting_set --app=$app --key=database --value=$database
 #=================================================
 # INSTALL DEPENDENCIES
 #=================================================
-ynh_script_progression --message="Installing dependencies..." --time --weight=1
+ynh_script_progression --message="Installing dependencies..."
```

Comment:
This section can be removed: dependencies are now installed with `extra_php_dependencies`

Reply:
Yes, but this can be used in future for different dependencies other then php modules. For eg. Redis 
Still this should be removed?